### ISSUE TITLE
PHP_CodeSniffer excludes src directory

### ISSUE THREAD

**Issue by pondermatic** [gocodebox/lifterlms#1947]
### Reproduction Steps

1. [Install WordPress locally](https://make.wordpress.org/core/handbook/tutorials/installing-wordpress-locally/) using a checkout from SVN or Git.
2. Clone the LifterLMS repo to `src/wp-content/plugins/lifterlms`.
3. Run PHP_CodeSniffer. `composer run check-cs-errors`

### Expected Behavior

There should be around 764 files sniffed.
```
............................................................  60 / 764 (8%)
```

### Actual Behavior

0 files are sniffed.

### Troubleshooting

The `<exclude-pattern>src/**/*</exclude-pattern>` line in commit bceea7d excludes all LifterLMS files if `/src/` is in the full path to LifterLMS.

If the intent is to exclude only JavaScript files anywhere in the `src` directory, then `<exclude-pattern>lifterlms/src/*.js</exclude-pattern>` works on my system.

If the intent is to exclude the entire `src` directory, then `<exclude-pattern>lifterlms/src/*</exclude-pattern>` works on my system.

I could not get the "relative" attribute in the [Annotated Ruleset wiki](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-Ruleset) to work.

### System and Environment Information

+ Windows 10
+ WordPress 5.8-src
+ LifterLMS dev-600
+ PHP_CodeSniffer 3.6.2

This issue has be recreated:
+ [x] Locally
+ [ ] On a staging site
+ [ ] On a production website
+ [ ] With only LifterLMS and a default theme

**Comment by thomasplevy** [comment]
We'll deal with this after we finish WP 5.9 / FSE and get 6.0 into at least an RC state

**Comment by thomasplevy** [comment]
@pondermatic is this fixed by #2130?

**Comment by pondermatic** [comment]
@thomasplevy, yes. Thanks for catching this. I forgot I had opened an issue about it.
